90zfsbootmenu/zfsbootmenu-help.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+# vim: softtabstop=2 shiftwidth=2 expandtab
+WIDTH="$( tput cols )"
+PREVIEW_SIZE=$(( WIDTH - 26 ))
```

Comment:
what happens if this becomes negative? Real question.

Reply:
Good catch!